operations/configuration_management/Differences_Between_the_V2_and_V3_CFS_APIs.md

```diff
@@ -3,32 +3,44 @@
+  configuration and inventory content from external repositories. See
+  [Using sources in a configuration layer](CFS_Configurations.md#using-sources-in-a-configuration-layer) and
+  [Using sources in additional inventory](Adding_Additional_Inventory.md#using-sources-in-additional-inventory).
 * Component records no longer include the `state` list of applied playbooks by default.
   The `state` can requested with the `state_details` parameter.
 * Some fields now have maximum sizes:
-    * Configuration names are now limited to 60 characters
-    * Additional inventory URLs are limited to 240 characters
-    * Configuration layer names are limited to 45 characters
-* The v3 API has three new global options: `default_page_size`, `debug_wait_time` and `include_ara_links`.
+    * Configuration names are now limited to 60 characters.
+    * Additional inventory URLs are limited to 240 characters.
+    * Configuration layer names are limited to 45 characters.
+* The v3 API has new global options:
+  [Additional inventory source](CFS_Global_Options.md#additional-inventory-source),
+  [Default page size](CFS_Global_Options.md#default-page-size),
+  [Debug wait time](CFS_Global_Options.md#debug-wait-time), and
+  [Include ARA links](CFS_Global_Options.md#include-ara-links).
   See [CFS Global Options](CFS_Global_Options.md) for more information.
-* Session and component records now include a `logs` field with a link to the ARA UI with the appropriate filter for that session or component.
+* Session and component records now include a `logs` field with a link to the ARA UI with the
+  appropriate filter for that session or component.
+  For more information, see [Ansible Log Collection](Ansible_Log_Collection.md) and
+  [Include ARA links](CFS_Global_Options.md#include-ara-links).
 * The `default_playbook` option is now deprecated.
   It can still be read using the v3 API but can not be set using the v3 API.
-  Any value set using the v2 API will still be usable by configurations, even v3 configurations, until the v2 API is removed.
-* Sessions now support a `debug_on_failure` option that will cause sessions that fail during Ansible execution to remain up for a limited time so that users can exec into the AEE container and debug the problem.
-  See [Troubleshoot CFS Issues](Troubleshoot_CFS_Issues.md) for more information.
+  Any value set using the v2 API will still be usable by configurations, even v3 configurations,
+  until the v2 API is removed.
+* Sessions now support a `debug_on_failure` option that will cause sessions that fail during Ansible
+  execution to remain up for a limited time so that users can `exec` into the
+  [Ansible Execution Environment (AEE)](Ansible_Execution_Environments.md) container and debug
+  the problem. See [Troubleshoot CFS Issues](Troubleshoot_CFS_Issues.md) for more information.
 * The CFS v3 supports new debugging playbooks which are included by default.
```

Comment:
"The CFS v3" is inconsistent with elsewhere in this document. Should it be "The v3 API"?